### DISCUSSION TITLE
I created a front end to whisper in python called `transcribe_anything`

### DISCUSSION THREAD

**Discussion by zackees** [openai/whisper#628] [Show and tell]
Check it out:

```bash
> pip install transcibe-anything
> transcribe_anything https://www.youtube.com/watch?v=YQVBPGj7gK4
```

It installs the dependencies of whisper ai. Right now only cpu is enabled but I'm working on getting the gpu device enabled.

Update: GPU/CUDA is now enabled, see https://github.com/zackees/transcribe-anything/blob/main/install_cuda.sh

You'll need to git clone the package to install the GPU/CUDA version of `transcribe_anything`, due to security reasons involved with `pip`/`pypi`.

Repo:
https://github.com/zackees/transcribe-anything

**Comment by dgoryeo** [comment]
Can the model be specified, sy --large? And language?

**Reply by zackees** [reply]
Yes, model selection, language and task type are all supported and passed to the whisper ai program.

Also now there is an install script that will install the gpu (cuda) version of torch to enable gpu acceleration, but it needs you to run `curl https://raw.githubusercontent.com/zackees/transcribe-anything/main/install_cuda.py | python` as there is no version of torch[gpu] on the pypi service (yet?).

**Reply by dgoryeo** [reply]
Unfortunately my GPU only has 4GB VRAM --meanwhile I want to use Whisper for Japanese that makes me to need the large model which needs 8+GB. I have to stick to CPU, or to move the execution to cloud/colab.
